src/components/QuickOpenModal.js

```diff
@@ -68,10 +70,18 @@ type GotoLocationType = {
 };
 
 function filter(values, query) {
-  return fuzzyAldrin.filter(values, query, {
+  let results = fuzzyAldrin.filter(values, query, {
     key: "value",
     maxResults: 1000
   });
+
+  if (prefs.projectDirectoryRoot) {
+    results = results.filter(result =>
+      result.url.includes(prefs.projectDirectoryRoot)
```

Comment:
How about `startsWith`?

Reply:
works for me

Reply:
We can't do `startsWith` because our projectDirectoryRoot is saved as `/davidwalsh.name/path/to/whatever` and the result.url is the true URL: `https://davidwalsh.name/path/to/whatever`.

Maybe we should save the root as true URL and that would simplify things.

Reply:
includes is good for now. i like keeping the tree data